packages/cfa-template/template/backend/serverless.yml

```diff
@@ -10,6 +10,7 @@ provider:
     environment:
         STAGE: ${opt:stage}
         AWS_NODEJS_CONNECTION_REUSE_ENABLED: 1
+        SECRET_ARN: { Ref: FriggEnvSecret }
     iamRoleStatements:
         - Effect: Allow
```

Comment:
Need to add:
```
- Effect: Allow
          Action:
                - secretsmanager:GetSecretValue
          Resource:
                - ${self:provider.environment.SECRET_ARN}
  ```
  
[  (seen here)](https://github.com/lefthookhq/42matters--frigg/blob/7da7c8f6c26a7cf151530449acbf2073da5ce974/backend/serverless.yml#L36C9-L40C58)